AmazonAppDev/react-native-multi-tv-app-sample
# feat: video player integration

**Feature: Video Player Integration**

This PR implements a video player for the app.

### Changes:
- [x] Added sample public videos and corresponding thumbnails.
- [x] Added a thumbnail on the `main` and `details` screens.
- [x] Added a video player to the `player` screen.
- [x] Implemented custom controls for Android TV and Android-based OSes, including play/pause, seek, back functions.
- [x] Kept native controls for Apple TV.
- [x] Working web support

### Testing:
- Prebuild the app.
- Run `yarn install` in the root directory.
- Use `yarn android --devices` and select a TV device.
- Install Pods
- Use `yarn ios`.

I've even tested it on Fire TV! :)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution under the terms of your choice.
